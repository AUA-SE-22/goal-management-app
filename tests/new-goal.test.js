import { render, screen } from '@testing-library/react'
import NewGoal from '../pages/goals/new-goal'

describe('Correctly renders', () => {

    it('the interface to add new goals', () => {
    const { container } = render(<NewGoal />)
    const button = screen.getByRole('button', {
    name: 'Add goal',
    })

    expect(button).toBeInTheDocument()
    expect(container).toMatchSnapshot()
    })
})