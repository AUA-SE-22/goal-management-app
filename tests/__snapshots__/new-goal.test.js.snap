// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders the interface to add new goals 1`] = `
<div>
  <div
    class="MuiBox-root css-yuob64"
  >
    <div
      class="MuiBox-root css-b7rhyr"
    >
      <div
        class="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
      >
        <div
          class="MuiFormControl-root css-1nrlq1o-MuiFormControl-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="false"
            for="title"
          >
            Goal Title
          </label>
          <div
            class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl css-1ptx2yq-MuiInputBase-root-MuiInput-root"
            required=""
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input MuiInput-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
              id="title"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="MuiFormControl-root css-1nrlq1o-MuiFormControl-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="false"
            for="description"
          >
            Description
          </label>
          <div
            class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
          >
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                id="description"
                required=""
                rows="5"
                type="text"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-ihdtdm"
                >
                  <span
                    class="notranslate"
                  >
                    â€‹
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          class="MuiFormControl-root css-1nrlq1o-MuiFormControl-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="submit"
          >
            Add goal
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
